src/main/java/com/bandwidth/iris/sdk/model/ImportTnOrder.java

```diff
@@ -67,6 +69,38 @@ public static OrderHistoryWrapper GetHistory( IrisClient client, String orderId
         return client.get( client.buildAccountModelUri( new String[] {IrisPath.IMPORT_TN_ORDERS, orderId, "history" } ), OrderHistoryWrapper.class);
     }
 
+    public static FileListResponse GetLoas( IrisClient client, String orderId ) throws Exception {
+        return client.get( client.buildAccountModelUri( new String[] {IrisPath.IMPORT_TN_ORDERS, orderId, IrisPath.LOAS } ), FileListResponse.class);
+    }
+
+    public static void PostLoasFile( IrisClient client, String orderId, File file, String contentType ) throws Exception {
```

Comment:
If it would make sense in the Java world, I'd prefer `UploadLoasFile` to be the method name